pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -560,32 +560,31 @@ def avg_time(cimtime, num_ops):
             number_of_operations = inst['NumberOfOperations']
 
             cimom_avg_elapsesd_time = avg_time(inst['CimomElapsedTime'],
-                                               number_of_operations)
+                                               number_of_operations) / 1000
 
             provider_avg_elapsesd_time = avg_time(inst['ProviderElapsedTime'],
-                                                  number_of_operations)
+                                                  number_of_operations) / 1000
 
             avg_request_size = avg_value(inst['RequestSize'],
                                          number_of_operations)
             avg_response_size = avg_value(inst['ResponseSize'],
                                           number_of_operations)
 
-            rows.append([number_of_operations,
-                         "{:.1f}".format(cimom_avg_elapsesd_time),
-                         "{:.1f}".format(provider_avg_elapsesd_time),
-                         "{:.1f}".format(avg_request_size),
-                         "{:.1f}".format(avg_response_size),
-                         op_type_name])
+            rows.append([op_type_name,
+                         number_of_operations,
```

Comment:
I originally put the operation as the last column for a reason, It is usually the widest column and it seemed to make reading easier if if did not take up a lot of the left side of the table.  That was maybe my preference and the way I read tables but since we are putting these things in limited width terminal envrionment it seemed logical.
**DISCUSSION**

Reply:
If the display is wider than the terminal width, each line spills over, and I find it easier in that case to locate the right row when it starts with the operation name.

Reply:
No issue. DONE